sdmetrics/reports/utils.py

```diff
@@ -169,19 +175,183 @@ def get_column_plot(real_column, synthetic_column, sdtype):
+        plot_bgcolor=BACKGROUND_COLOR,
+    )
+
+    return fig
+
+
+def make_discrete_column_pair_plot(real_data, synthetic_data):
+    """Make a column pair plot for discrete data.
+
+    Args:
+        real_data (pandas.DataFrame):
+            The real data for the desired column pair.
+        synthetic_column (pandas.Dataframe):
+            The synthetic data for the desired column pair.
+
+    Returns:
+        plotly.graph_objects._figure.Figure
+    """
+    columns = real_data.columns
+    real_data = real_data.copy()
+    real_data['Data'] = 'Real'
+    synthetic_data = synthetic_data.copy()
+    synthetic_data['Data'] = 'Synthetic'
+    all_data = pd.concat([real_data, synthetic_data], axis=0, ignore_index=True)
+
+    fig = px.density_heatmap(
+        all_data,
+        x=columns[0],
+        y=columns[1],
+        facet_col='Data',
+        histnorm='probability'
+    )
+
+    fig.update_layout(
+        title_text=f"Real vs Synthetic Data for columns '{columns[0]}' and '{columns[1]}'",
+        coloraxis={'colorscale': [DATACEBO_DARK, DATACEBO_LIGHT]},
+    )
+
+    fig.for_each_annotation(lambda a: a.update(text=a.text.split('=')[-1] + ' Data'))
 
     return fig
 
 
+def make_mixed_column_pair_plot(real_data, synthetic_data):
+    """Make a column pair plot for mixed discrete and continuous column data.
+
+    Args:
+        real_data (pandas.DataFrame):
+            The real data for the desired column pair.
+        synthetic_column (pandas.Dataframe):
+            The synthetic data for the desired column pair.
+
+    Returns:
+        plotly.graph_objects._figure.Figure
+    """
+    columns = real_data.columns
+    real_data = real_data.copy()
+    real_data['Data'] = 'Real'
+    synthetic_data = synthetic_data.copy()
+    synthetic_data['Data'] = 'Synthetic'
+    all_data = pd.concat([real_data, synthetic_data], axis=0, ignore_index=True)
+
+    fig = px.box(
+        all_data,
+        x=columns[0],
+        y=columns[1],
+        color='Data',
+        color_discrete_map={'Real': DATACEBO_DARK, 'Synthetic': DATACEBO_LIGHT},
+    )
+
+    fig.update_layout(
+        title=f"Real vs. Synthetic Data for columns '{columns[0]}' and '{columns[1]}'",
+        plot_bgcolor=BACKGROUND_COLOR
+    )
+
+    return fig
+
+
+def get_column_pair_plot(real_data, synthetic_data, sdtypes):
+    """Return a plot of the real and synthetic data for a given column pair.
+
+    Args:
+        real_data (pandas.DataFrame):
+            The real data for the desired column pair.
+        synthetic_column (pandas.Dataframe):
+            The synthetic data for the desired column pair.
+        sdtypes (list[string]):
+            The data type of the column pair. The data type string must be one of
+            ('numerical', 'datetime', 'categorical', or 'boolean').
+
+    Returns:
+        plotly.graph_objects._figure.Figure
+    """
+    all_sdtypes = CONTINUOUS_SDTYPES + DISCRETE_SDTYPES
+    if not sdtypes[0] in all_sdtypes:
```

Comment:
It's preferable to use `x not in` than `not x in`, so: `if sdtypes[0] not in all_sdtypes:`